include/CIndexStoreDB/CIndexStoreDB.h

```diff
@@ -215,6 +217,27 @@ indexstoredb_delegate_event_get_kind(_Nonnull indexstoredb_delegate_event_t);
 INDEXSTOREDB_PUBLIC
 uint64_t indexstoredb_delegate_event_get_count(_Nonnull indexstoredb_delegate_event_t);
 
+/// Valid only if the event kind is \p INDEXSTOREDB_EVENT_UNIT_OUT_OF_DATE, otherwise returns null.
+INDEXSTOREDB_PUBLIC _Nullable indexstoredb_unit_info_t
+indexstoredb_delegate_event_get_outofdate_unit_info(_Nonnull indexstoredb_delegate_event_t);
+
+/// Returns number of nanoseconds since clock's epoch.
+/// Valid only if the event kind is \p INDEXSTOREDB_EVENT_UNIT_OUT_OF_DATE, otherwise returns 0.
+INDEXSTOREDB_PUBLIC uint64_t
+indexstoredb_delegate_event_get_outofdate_modtime(_Nonnull indexstoredb_delegate_event_t);
+
+/// Valid only if the event kind is \p INDEXSTOREDB_EVENT_UNIT_OUT_OF_DATE, otherwise returns false.
+INDEXSTOREDB_PUBLIC bool
+indexstoredb_delegate_event_get_outofdate_is_synchronous(_Nonnull indexstoredb_delegate_event_t);
+
+/// Valid only if the event kind is \p INDEXSTOREDB_EVENT_UNIT_OUT_OF_DATE, otherwise returns null.
+INDEXSTOREDB_PUBLIC const char * _Nullable
+indexstoredb_delegate_event_get_outofdate_trigger_original_file(_Nonnull indexstoredb_delegate_event_t);
```

Comment:
Doc comment should mention the ownership of the string

```swift
/// The string has the same lifetime as the \c indexstoredb_delegate_event_t.
```

Same for `indexstoredb_delegate_event_get_outofdate_trigger_description`.

Reply:
Done.